recipe/meta.yaml

```diff
@@ -42,7 +42,9 @@ requirements:
     - torch_em >=0.7.1
     - tqdm
     - timm
-    - zarr
+    # 2.18.4 shows a circular import
+    # We still are not compatible with zarr 3
+    - zarr <3.0,!=2.18.4
```

Comment:
@constantinpape we found that we had trouble if zarr's version was too high.

Can you confirm?

Reply:
Yes, that makes sense. We definitely need to exclude zarr < 3.0. I wasn't aware of the issue with 2.18.4, but if you encountered this then it's good to exclude it.